### DISCUSSION TITLE
Missing viewbox attribute

### DISCUSSION THREAD

**Discussion by mgmmgm** [yoriiis/svg-chunk-webpack-plugin#25] [Q&A]
Hi,
We are trying to use your awesome plugin but encountered a small issue.
Viewbox attribute on the original svg is missing from the generated symbol element.
This is our plugin configuration:
new SvgChunkWebpackPlugin({filename: 'sprite.[contenthash].svg', svgstoreConfig: {renameDefs: true, copyAttrs: true}}),

Any ideas?

**Comment by yoriiis** [comment]
Hello @mgmmgm, thanks for your feedback!
Could you share a reproductible repository please? Thanks
